src/app/workspace/resource/services/resource.service.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { TestBed } from '@angular/core/testing';
+import { AppInitService } from 'src/app/app-init.service';
+import { ResourceService } from './resource.service';
+
+describe('ResourceService with iriBase = http://rdfh.ch', () => {
+    let service: ResourceService;
+
+    const appInitSpy = {
+        dspAppConfig: {
+            iriBase: 'http://rdfh.ch'
+        }
+    };
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            providers: [
+                {
+                    provide: AppInitService,
+                    useValue: appInitSpy
+                }
+            ]
+        });
+        service = TestBed.inject(ResourceService);
+    });
+
+    it('should be created', () => {
+        expect(service).toBeTruthy();
+    });
+
+    it('should return the resource iri from project code and resource uuid', () => {
+        const iri = service.getResourceIri('082B', 'SQkTPdHdTzq_gqbwj6QR-A');
+        expect(iri).toEqual('http://rdfh.ch/082B/SQkTPdHdTzq_gqbwj6QR-A');
+    });
+
+    it('should return the app\'s resource path from project iri', () => {
+        const path = service.getResourcePath('http://rdfh.ch/082B/SQkTPdHdTzq_gqbwj6QR-A');
+        expect(path).toEqual('/082B/SQkTPdHdTzq_gqbwj6QR-A');
+    });
+});
+
+// if the iriBase has an slash at the end
+describe('ResourceService with iriBase = http://rdfh.ch/', () => {
+    let service: ResourceService;
+
+    const appInitSpy = {
+        dspAppConfig: {
+            iriBase: 'http://rdfh.ch/'
+        }
+    };
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            providers: [
+                {
+                    provide: AppInitService,
+                    useValue: appInitSpy
+                }
+            ]
+        });
+        service = TestBed.inject(ResourceService);
+    });
+
+    it('should be created', () => {
+        expect(service).toBeTruthy();
```

Comment:
you can move this `expect` to line 61 and remove this `it` since it's a bit better to ensure the service is created correctly during each `BeforeEach()`. You can do the same with the `it` on line 26

Reply:
Done in c89eef9